src/inc/quic_trace.h

```diff
@@ -103,6 +103,7 @@ typedef enum QUIC_TRACE_API_TYPE {
     QUIC_TRACE_API_DATAGRAM_SEND,
     QUIC_TRACE_API_CONNECTION_COMPLETE_RESUMPTION_TICKET_VALIDATION,
     QUIC_TRACE_API_CONNECTION_COMPLETE_CERTIFICATE_VALIDATION,
+    QUIC_TRACE_API_STREAM_PROVIDE_RECEIVE_BUFFERS,
```

Comment:
Our ETW plugin will need to be updated too.

Reply:
Will update these (and SpinQuic) in separate PRs